src/tools/vfs/vfs_main.c

```diff
@@ -342,26 +342,39 @@ int vfs_start()
 {
     int ret;
 
+    ret = mkdir(g_opts.mountpoint_dir, S_IRWXU);
+    if (ret < 0) {
+        vfs_error("could not create directory '%s': %m", g_opts.mountpoint_dir);
```

Comment:
1. should not create the directory if already exists
2. should remove directory only if actually created it
another option:
don't create directory at all. print error that directory must exist prior to starting mount daemon.

Reply:
1. It doesn't create the directory if already exists.

```
$ mkdir <path>
$ ucx_vfs start -d ~/vfs -f
Error: could not create directory '<path>': File exists
```

2. Thus the daemon removes directory only previously created by itself

Reply:
1. i meant the daemon will work but just not create the dir. because after this PR you always have to run it with a path that does not already exist. but before this PR you could give an existing directory